README.md

```
@@ -1,3 +1,76 @@
+# Build and test clob contract
+cd ../clob
+cargo build
+cargo test
+cargo fmt --all -- --check
+cargo clippy --all-targets --all-features -- -D clippy::all -D warnings
+cargo build --target wasm32-unknown-unknown --release
+```
+
+### Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)